From faa4d9443e70182fb222a45a878f209de0d974a3 Mon Sep 17 00:00:00 2001
From: Barry Song <Baohua.Song@csr.com>
Date: Tue, 27 Sep 2011 22:05:44 +0200
Subject: PM / Hibernate: Fix typo in a kerneldoc comment

Fix a typo in a function name in the kerneldoc comment next to
resume_target_kernel().

[rjw: Changed the subject slightly, added the changelog.]

Signed-off-by: Barry Song <Baohua.Song@csr.com>
Signed-off-by: Rafael J. Wysocki <rjw@sisk.pl>
(cherry picked from commit 21e82808fc465b66fedaac0f4e885cafb304e843)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 kernel/power/hibernate.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/power/hibernate.c b/kernel/power/hibernate.c
index 878218e..089ab9c 100644
--- a/kernel/power/hibernate.c
+++ b/kernel/power/hibernate.c
@@ -467,7 +467,7 @@ static int resume_target_kernel(bool platform_mode)
  * @platform_mode: If set, use platform driver to prepare for the transition.
  *
  * This routine must be called with pm_mutex held.  If it is successful, control
- * reappears in the restored target kernel in hibernation_snaphot().
+ * reappears in the restored target kernel in hibernation_snapshot().
  */
 int hibernation_restore(int platform_mode)
 {
-- 
1.7.10.1.362.g242cab3

